torchvision/transforms/v2/functional/_augment.py

```diff
@@ -53,3 +56,40 @@ def erase_video(
     video: torch.Tensor, i: int, j: int, h: int, w: int, v: torch.Tensor, inplace: bool = False
 ) -> torch.Tensor:
     return erase_image(video, i=i, j=j, h=h, w=w, v=v, inplace=inplace)
+
+
+def jpeg(image: torch.Tensor, quality: int) -> torch.Tensor:
+    """See :class:`~torchvision.transforms.v2.JPEG` for details."""
+    # TODO: this is probably not torchscript-compatible?
+    if torch.jit.is_scripting():
+        return jpeg_image(image, quality=quality)
+
+    _log_api_usage_once(jpeg)
+
+    kernel = _get_kernel(jpeg, type(image))
+    return kernel(image, quality=quality)
+
+
+@_register_kernel_internal(jpeg, torch.Tensor)
+@_register_kernel_internal(jpeg, tv_tensors.Image)
+@_register_kernel_internal(jpeg, tv_tensors.Video)
+def jpeg_image(image: torch.Tensor, quality: int) -> torch.Tensor:
+    original_shape = image.shape
+    image = image.reshape((-1,) + image.shape[-3:])
```

Comment:
I know reshape shoudn't involve a copy here but just to be on the safe side, let's use `view`? (same below)

```suggestion
    image = image.view((-1,) + image.shape[-3:])
```